cmd/immu/immu.go

```diff
@@ -96,7 +97,11 @@ func main() {
 				if err != nil {
 					return err
 				}
-				printItem([]byte(args[0]), value, response)
+
+				response, err := immuClient.Connected(func() (interface{}, error) {
```

Comment:
Just to confirm/double-check: this was added intentionally, so that `set` will return the complete item (with `get`), not just the index returned by `Set(SV)` (as it was before), right?

Reply:
Since timestamp is generated inside the Client, but according requirements Set methods should return only index. So in immu CLI we should perform Get after Set to read generated timestamp.